pkg/manifest/manifest.go

```diff
@@ -5,38 +5,36 @@
 package manifest
 
 import (
+	"encoding"
 	"errors"
 	"net/http"
 
 	"github.com/ethersphere/bee/pkg/swarm"
 )
 
+// ErrNotFound is returned when an Entry is not found in the manifest
 var ErrNotFound = errors.New("manifest: not found")
 
-// Parser for manifest
-type Parser interface {
-	// Parse parses the encoded manifest data and returns the result
-	Parse(bytes []byte) (Interface, error)
-}
-
 // Interface for operations with manifest
 type Interface interface {
-	// Add a manifest entry to specified path
+	// Add a manifest entry to the specified path
 	Add(string, Entry)
-	// Remove reference from file on specified path
+	// Remove a manifest entry on the specified path
 	Remove(string)
-	// FindEntry returns manifest entry if one is found on specified path
+	// FindEntry returns a manifest entry if one is found in the specified path
 	FindEntry(string) (Entry, error)
-	// Serialize return encoded manifest
-	Serialize() ([]byte, error)
+	// BinaryMarshaler is the interface implemented by an object that can marshal itself into a binary form
+	encoding.BinaryMarshaler
+	// BinaryUnmarshaler is the interface implemented by an object that can unmarshal a binary representation of itself
+	encoding.BinaryUnmarshaler
 }
 
-// Entry represents single manifest entry
+// Entry represents a single manifest entry
 type Entry interface {
-	// GetReference returns address of the entry file
+	// GetReference returns the address of the file in the entry
```

Comment:
I think we can remove the `Get` prefixes here too. Just as `swarm.Address` has `String()` and not `GetString()`, or `Bytes()` and not `GetBytes()`

Reply:
fixed, thanks